Dockerfile

```diff
@@ -56,4 +56,6 @@ RUN R CMD javareconf \
     && printf "GITHUB_PAT=$GITHUB_PAT\n" > .Renviron \
     && /usr/local/lib/R/bin/Rscript /tmp/init.R \
     && rm -f /tmp/init.R \
-    && rm -f .Renviron
+    && rm -f .Renviron \
+	&& chown :users -R /usr/local/lib/R/site-library \
```

Comment:
Jako mohl si aspon zachovat stejne odsazeni jako ve zbytku souboru :)

Reply:
No to jsem teda mohl :)